CHANGELOG.md

```diff
@@ -33,6 +33,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - Change font size and color for text.[#975](https://github.com/rokwire/events-manager/issues/975)
 - Change the CSS class names to match the HTML controls. [#973](https://github.com/rokwire/events-manager/issues/973)
+- Change the GUI of text input field box. [#991](https://github.com/rokwire/events-manager/issues/991)
```

Comment:
Please move this entry to the Unreleased section.

Reply:
fixed.